kava-stats.js

```diff
@@ -522,10 +525,28 @@ const mapSupplyAndMarket = async (denoms, siteData) => {
   return coins
 }
 
-const mapUsdxMarketData = async (usdxMarketJson) => {
```

Comment:
Renamed this function since we will now use it to pull the data we need from CoinGecko's API for price change for both USDX and SWP